components/prize_pool/prize_pool_award_placement.lua

```diff
@@ -0,0 +1,118 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:PrizePool/Award/Placement
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local BasePlacement = Lua.import('Module:PrizePool/Placement/Base', {requireDevIfEnabled = true})
+
+local Opponent = require('Module:OpponentLibraries').Opponent
+
+local _tbd_index = 0
+
+local PRIZE_TYPE_BASE_CURRENCY = 'BASE_CURRENCY'
+local PRIZE_TYPE_POINTS = 'POINTS'
+
+--- @class AwardPlacement
+--- An AwardPlacement is a set of opponents who all share the same award in the tournament.
+--- Its input is generally a table created by `Template:Slot`.
+local AwardPlacement = Class.new(BasePlacement)
+
+--- @param award string Award of this slot/placement
+function AwardPlacement:create(award)
+	self.award = award
+	self.count = tonumber(self.args.count)
+	self.opponents = self:parseOpponents(self.args)
+	self.count = self.count or math.max(#self.opponents, 1)
```

Comment:
Why count twice?

Reply:
because it is needed in `parseOpponents` if it is a valid input (i.e. number)
and the 2nd one sets it in case it was not set